pypeit/tests/test_datasec.py

```diff
@@ -1,34 +0,0 @@
-"""
-Module to run tests on ampsec definition
-"""
-import pytest
-
-import os
-import numpy as np
-
-from pypeit.core import pixels
-from pypeit.core import procimg
-from pypeit.spectrographs.util import load_spectrograph
-
-@pytest.fixture
-def spectrograph():
-    return load_spectrograph('shane_kast_blue')
-
-def data_path(filename):
-    data_dir = os.path.join(os.path.dirname(__file__), 'files')
-    return os.path.join(data_dir, filename)
-
-def test_ampsec(spectrograph):
```

Comment:
Are there equivalent tests for where datasec image is currently generated?

Reply:
not sure